bazel/markdown_to_html/markdown_to_html.bzl

```diff
@@ -0,0 +1,74 @@
+load("@build_bazel_rules_nodejs//:providers.bzl", "run_node")
+
+def _relative_to_label(label, short_path):
```

Comment:
Nit: I'd probably put that into `bazel/private`

Reply:
Done